src/gists/gist.jl

```diff
@@ -0,0 +1,79 @@
+type Gist <: GitHubType
+    url::Nullable{HttpCommon.URI}
+    forks_url::Nullable{HttpCommon.URI}
+    commits_url::Nullable{HttpCommon.URI}
+    id::Nullable{GitHubString}
+    description::Nullable{GitHubString}
+    public::Nullable{Bool}
+    owner::Nullable{Owner}
+    user::Nullable{Owner}
+    truncated::Nullable{Bool}
+    comments::Nullable{Int}
+    comments_url::Nullable{HttpCommon.URI}
+    html_url::Nullable{HttpCommon.URI}
+    git_pull_url::Nullable{HttpCommon.URI}
+    git_push_url::Nullable{HttpCommon.URI}
+    created_at::Nullable{Dates.DateTime}
+    updated_at::Nullable{Dates.DateTime}
+    forks::Nullable{Vector{Gist}}
+    files::Nullable{Dict}
+    history::Nullable{Vector{Dict}}
```

Comment:
Ah, I read this too quickly, sorry. Yeah, if you wanted to make that conversion, you could place it in the constructor (which unfortunately will probably have a pretty verbose signature). 

You could also have a new GitHubType describing the objects contained in `history`. 

I also think it'd be okay to not make the conversion at all - it's your call.